### DISCUSSION TITLE
Consulta Voluntario

### DISCUSSION THREAD

**Discussion by suumoo5** [dds-jv/foro#255] [TPA - 2021] [answered]
Buenas, tenemos una duda sobre Voluntario. En el enunciado dice: 
"_Se debe permitir generar usuarios Voluntarios, los cuales podrán aprobar y gestionar las
publicaciones de mascotas perdidas._"

Eso indica que cada voluntario puede aprobar/rechazar **cualquier** publicacion? O solamente las que estén asociadas a su asociación en particular?

**Comment by gastonprieto** [comment, marked as answer]
Buenas @suumoo5 

Tendría sentido que solo la de las organizaciones que pertenece. Pero no es una validación importante para esta entrega.

**Reply by gastonprieto** [reply]
te sirvio @suumoo5 ?

**Reply by suumoo5** [reply]
Si, gracias, con eso que me dijiste, y lo que le pregunté a mi ayudante, pudimos seguir un diseño que haga que un voluntario solo pueda aprobar/rechazar las publicaciones de su asociación.
